docs/concepts/model_bundles.md

```diff
@@ -114,31 +141,73 @@ module path to the `load_predict_fn`.
     class MyResponseSchema(BaseModel):
         __root__: int
      
-    ENV_PARAMS = {
-        "framework_type": "pytorch",
-        "pytorch_image_tag": "1.7.1-cuda11.0-cudnn8-runtime",
-    }
-
     BUNDLE_PARAMS = {
         "model_bundle_name": "test-bundle",
         "base_paths": [directory],
-        "requirements_path": "requirements.txt",
-        "env_params": ENV_PARAMS,
-        "load_predict_fn": "predict.my_load_predict_fn",
+        "load_predict_fn_module_path": "predict.my_load_predict_fn",
         "load_model_fn_module_path": "model.my_load_model_fn",
         "request_schema": MyRequestSchema,
         "response_schema": MyResponseSchema,
+        "requirements_path": "requirements.txt",
+        "pytorch_image_tag": "1.7.1-cuda11.0-cudnn8-runtime",
     }
 
     client = LaunchClient(api_key=os.getenv("LAUNCH_API_KEY"))
-    client.create_model_bundle_from_dirs(**BUNDLE_PARAMS)
+    client.create_model_bundle_from_dirs_v2(**BUNDLE_PARAMS)
 
     # Clean up files from demo
     os.remove(model_filename)
     os.remove(predict_filename)
     os.rmdir(directory)
     ```
 
+=== "Creating From a Runnable Image"
+    ```py
+    import os
+    from pydantic import BaseModel
+    from launch import LaunchClient
+
+
+    class MyRequestSchema(BaseModel):
+        x: int
+        y: str
+
+    class MyResponseSchema(BaseModel):
+        __root__: int
+
+
+    BUNDLE_PARAMS = {
+        "model_bundle_name": "test-bundle",
+        "load_model_fn": my_load_model_fn,
+        "load_predict_fn": my_load_predict_fn,
+        "request_schema": MyRequestSchema,
+        "response_schema": MyResponseSchema,
+        "repository": "launch_rearch",
+        "tag": "12b9131c5a1489c76592cddd186962cce965f0f6-cpu",
+        "command": [
+            "dumb-init",
+            "--",
+            "ddtrace-run",
+            "run-service",
+            "--config",
+            "/install/launch_rearch/config/service--user_defined_code.yaml",
+            "--concurrency",
+            "1",
+            "--http",
+            "production",
+            "--port",
+            "5005",
+        ],
+        "env": {
+            "TEST_KEY": "test_value",
+        },
```

Comment:
I would try to use a different example here @phil-scale -- we could just have stand-in values, right?

Reply:
No these are actually run in integration tests. If you have another example that would work, I'm happy to swap to them.

Reply:
Otherwise yes I definitely expect to replace these with a more demonstrative example in the future.